lib/ui/painting/canvas.cc

```diff
@@ -242,6 +243,24 @@ void Canvas::drawCircle(double x,
   canvas_->drawCircle(x, y, radius, *paint.paint());
 }
 
+void Canvas::drawArc(double left,
+             double top,
+             double right,
+             double bottom,
+             double startAngle,
+             double sweepAngle,
+             bool useCenter,
+             const Paint& paint,
+             const PaintData& paint_data) {
+  if (!canvas_)
+    return;
+  canvas_->drawArc(SkRect::MakeLTRB(left, top, right, bottom),
+                   startAngle * 180.0 / M_PI,
```

Comment:
I don't think this layer is meant to have or enforce opinions about argument types. Should just forward the arguments as is to Skia. We can perform these conversions at a higher level.